runtime/src/append_vec.rs

```diff
@@ -56,14 +61,23 @@ impl<'a> StoredAccount<'a> {
     }
 }
 
+#[derive(Debug)]
+#[allow(clippy::mutex_atomic)]
 pub struct AppendVec {
+    path: PathBuf,
     map: MmapMut,
     // This mutex forces append to be single threaded, but concurrent with reads
     append_offset: Mutex<usize>,
     current_len: AtomicUsize,
     file_size: u64,
 }
 
+impl Drop for AppendVec {
+    fn drop(&mut self) {
+        let _ = std::fs::remove_dir_all(&self.path.parent().unwrap());
```

Comment:
convention for this is _ignored =

Reply:
will update this

> convention for this is _ignored =

will update accordingly